test/e2e/util.go

```diff
@@ -2141,3 +2141,36 @@ func OpenWebSocketForURL(url *url.URL, config *client.Config, protocols []string
 	cfg.Protocol = protocols
 	return websocket.DialConfig(cfg)
 }
+
+// getIngressAddress returns the ips/hostnames associated with the Ingress.
+func getIngressAddress(client *client.Client, ns, name string) ([]string, error) {
```

Comment:
It's actually just returning whatever the loadbalancer status has, which is a string because it could either be an IP or hostname and there's no type that captures both properly https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L1237